CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -15,6 +15,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 - Compile binaries using [go 1.21.12](https://github.com/etcd-io/etcd/pull/18271).
 - [Fully address CVE-2023-45288 and fix govulncheck CI check](https://github.com/etcd-io/etcd/pull/18170)
 
+### Package clientv3
+- [Print gRPC metadata in guarnteed order using the official go fmt pkg](https://github.com/etcd-io/etcd/pull/18312).
+
```

Comment:
Minor comment: put this above section `Dependencies`

Reply:
Thanks! updated